Sources/Diff/DeepDiff/CollectionViewDeepDiffAdaptor.swift

```diff
@@ -27,7 +34,7 @@ final class CollectionViewDeepDiff: CollectionViewDiff {
     }
 }
 
-public final class DeepDiffDiffableItemWrapper<T: DiffItem>: DiffAware {
+final class DeepDiffDiffableItemWrapper<T: DiffItem>: DiffAware {
```

Comment:
Name `DeepDiffDiffableItemWrapper` looks ugly. But if we remove `Diffable` subword it will become better